### ISSUE TITLE
Invalid package.json

### ISSUE THREAD

**Issue by CrypticSwarm** [SnapluDJ/Multi-room-Chat-Server#1]
The package.json is from a different project. Specifically `node-uuid`.

This file should specify any dependencies such as `socket.io`. It would also be convenient to specify a valid `main`.

**Comment by SnapluDJ** [comment]
Yes, you are right, I looked up the same project in BitBucket and didn't find the package.json file in it. Based on the files in the repository, it should belongs to another project, but I can not figure out which project now.

**Comment by CrypticSwarm** [comment]
Looks like it is from (http://github.com/broofa/node-uuid), which you is likely the dependency used (https://github.com/SnapluDJ/Multi-room-Chat-Server/blob/master/prototype.js#L4).
